### DISCUSSION TITLE
Adding Night light

### DISCUSSION THREAD

**Discussion by swapnilxi** [MonitorControl/MonitorControl#1587] [General]
### Before opening the issue, have you...?

- [X] Searched for existing issues

### Is your feature request related to a problem? Please describe

yes when I use other apps for night light , monintor control does not work

### Describe the solution you'd like

just like brightness , I want inbuilt night light

### Describe alternatives you've considered

Flux

**Comment by waydabber** [comment]
Temperature control won't be added to MonitorControl but it has been added to [BetterDisplay](https://github.com/waydabber/BetterDisplay#readme) (under Image Adjustments).
